source/_patterns/config/_gesso-theme.scss

```diff
@@ -0,0 +1,238 @@
+$gesso: (
```

Comment:
What is this file?

Reply:
Artifact from before you restructured the file organization. Removing